packages/babel-plugin-transform-modules-commonjs/test/fixtures/rewritethis-computed-class-elements/input.mjs

```diff
@@ -0,0 +1 @@
+export class C { [this.name] = 42 }
```

Comment:
This needs to be inside a nested folder; otherwise, the file is executed instead of being transformed and compared with `output.js`.

Can you move it inside `packages/babel-plugin-transform-modules-commonjs/test/fixtures/misc/undefined-this-computed-class-key/`?